src/Microsoft.Diagnostics.Runtime/src/Linux/ElfLoadedImage.cs

```diff
@@ -12,7 +13,7 @@ namespace Microsoft.Diagnostics.Runtime.Utilities
     /// </summary>
     public class ElfLoadedImage
     {
-        private readonly List<ElfFileTableEntryPointers64> _fileTable = new(4);
+        private readonly SortedList<ulong, ElfFileTableEntryPointers64> _fileTable = new(4);
```

Comment:
We don't actually need a sorted list do we?  If Is _fileTable used anywhere else?

What about `private int _minimumPointer;`.

In AddTableEntryPointers:  `if (_minimumPointer == 0 || pointers.Start < _minimumPointer) _minimumPointer = pointers.Start;`

And BaseAddress can be replaced by:

    private ulong _baseAddress;
    public BaseAddress { get => _baseAddress == 0 ? _minimumPointer : _baseAddress; } set => _baseAddress = value; }

Or something similar?  If we need to keep a sorted list of file table pointers for other reasons then this is fine as-is, but there's no reason to keep a sorted list just to find the minimum address.